.github/workflows/linux_qemu.yml

```diff
@@ -107,12 +115,20 @@ jobs:
 
     - name: Initialize binfmt_misc for qemu-user-static
       run: |
-        docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+        # For Loongarch64, a specific path needs to be specified to obtain qemu-user-static
+        if [ "${{ matrix.BUILD_PROP[0] }}" = "loongarch64" ]; then
+          docker run --rm --privileged loongcr.lcpu.dev/multiarch/archlinux --reset -p yes
+        else
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+        fi
 
     - name: Install GCC cross-compilers
       run: |
-        sudo apt update
-        sudo apt install -y ninja-build gcc-${TOOLCHAIN_NAME} g++-${TOOLCHAIN_NAME} gfortran-${TOOLCHAIN_NAME}
+        # For LoongArch64, Directly use the toolchain integrated in Docker
```

Comment:
The LoongArch cross-toolchain and QEMU cannot be obtained directly from the source on Ubuntu 22.04 unless the test environment is upgraded to Ubuntu 24.04. To avoid impacting tests on other platforms, QEMU testing on LoongArch is currently controlled through branching. As you mentioned the downside, we will create a separate testing task for LoongArch specifically on Ubuntu 24.04.

Reply:
I have modified it.